src/promptflow-evals/promptflow/evals/_user_agent.py

```diff
@@ -0,0 +1,6 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from promptflow.evals._version import VERSION
+
+USER_AGENT = "{}/{}".format("promptflow-evals", VERSION)
```

Comment:
As a side note, we should send user agent to RAI service too if we don't do that today.

Reply:
Will follow up on a separate PR since there is a separate PR for an associated work.